### PR TITLE
Added support for contact images

### LINKED ISSUE #33: Add front end support for uploading a picture
**Time Estimate:** 3h

### LINKED ISSUE #54: Incorporate viewing pictures into front-end
**Time Estimate:** 2h

### PR DESCRIPTION
**Pull request by AlexMaclean**
Added front-end support for viewing contact images in the contact list and detailed contact view modal.

closes #54
closes #33

### CODE REVIEW

File: react-frontend/src/ContactList.js [hunk 1]
```diff
@@ -150,6 +164,25 @@ const LabelList = (props) => {
   return <>{labels}</>;
 };
 
+const getImageUrl = (image) => {
+  if (image && image.type && image.url) {
+    if (image.type === "external") {
+      return image.url;
+    }
+  }
+  return "/contact-no-image.png";
```
Comment (baileywickham) [inline react-frontend/src/ContactList.js:173]: What is displayed in this case?

**Comment by AlexMaclean** [comment]
I've completed the front-end portion of image support for the app.

@ysunkara @baileywickham @cprghunter Please review this PR in the next couple days.

Thx!

**Review by baileywickham — APPROVED** [review]
Looks good to me!

**Review by ysunkara — APPROVED** [review]
Looks good to me

File: react-frontend/src/index.css [hunk 2]
```diff
@@ -92,3 +92,17 @@ body {
 .sidebar-heading {
   text-transform: uppercase;
 }
+
+.table-image {
+  width: 30px;
```
Comment (cprghunter) [inline react-frontend/src/index.css:97]: Is there a better way in JS to handle these kinds of "magic numbers" so things like image size and border color are more easily configurable?

Comment (cprghunter) [reply react-frontend/src/index.css:97]: If not, LGTM!

Comment (AlexMaclean) [reply react-frontend/src/index.css:97]: Generally the way to abstract away from magic numbers like this in the CSS file would be to use Sass, however given the size of this project I don't think that is necessary. If we want to use that in the future, its an option, but outside of the scope of this feature.

**Comment by AlexMaclean** [comment]
Thanks all I'm merging these changes in now

**Pull request merged**